libs/@guardian/libs/src/consent-management-platform/onConsentChange.ts

```diff
@@ -90,8 +96,12 @@ const getConsentState: () => Promise<ConsentState> = async () => {
 	switch (getCurrentFramework()) {
 		case 'aus':
 			return enhanceConsentState({ aus: await getAUSConsentState() });
-		case 'ccpa':
-			return enhanceConsentState({ ccpa: await getCCPAConsentState() });
+		case 'usnat': {
+			const usnatConsentState = await getUSNATConsentState();
+			return enhanceConsentState({
+				usnat: usnatConsentState,
+			});
```

Comment:
Minor but is there any reason to _not_ follow the pattern used for the other frameworks where the await is inlined? It might be surprising to future readers.